app/views/contacts/index.html.erb

```diff
@@ -1,45 +1,89 @@
-      <th>Twitter</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @contacts.each do |contact| %>
-      <tr>
-        <td><%= contact.name %></td>
-        <td><%= contact.last_name %></td>
-        <td><%= contact.phone_number %></td>
-        <td><%= contact.communication_channel %></td>
-        <td><%= contact.user_id %></td>
-        <td><%= contact.email %></td>
-        <td><%= contact.organization %></td>
-        <td><%= contact.facebook %></td>
-        <td><%= contact.instagram %></td>
-        <td><%= contact.twitter %></td>
-        <td><%= link_to 'Show', contact %></td>
-        <td><%= link_to 'Edit', edit_contact_path(contact) %></td>
-        <td><%= link_to 'Destroy', contact, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
+<h2 class="h2 offset-1">Your contacts 
+  <span class="badge">
+    <%= link_to new_contact_path, class: "btn btn-link btn-sm text-muted" do %>
+    <i class="bi bi-person-plus-fill" style="font-size: 2rem"></i>
     <% end %>
-  </tbody>
-</table>
+  </span>  
+</h2>
+
+<div class="container">
+  <%= search_form_for @q do |f| %>
+    <h4>Search for a contact:</h4>
+    
+      <div class="row justify-content-left offset-1">
+        <div class="col-md-3">
+          <%= f.label :name_cont, "Name containing" %>
+          <%= f.text_field :name_cont, :placeholder => "Enter a few characters" %>
+        </div>
+
+        <div class="col-md-3">
+          <%= f.label :last_name_cont, "Last name containing" %>
+          <%= f.text_field :last_name_cont, :placeholder => "Enter a few characters" %>
+        </div>
+
+        <div class="col-md-3">
+          <%= f.label :organization_cont, "Organization containing" %>
+          <%= f.text_field :organization_cont, :placeholder => "Enter a few characters" %>
+        </div>
+
+        <div class="col-md-3">
+          <%= f.label :phone_number_cont, "Phone number containing" %>
+          <%= f.text_field :phone_number_cont, :placeholder => "Enter a few characters" %>
+        </div>
+      </div>
+    
+      <div class="row mt-3 justify-content-left offset-1">
+        <%= f.submit :class => "btn btn-primary btn-block" %>
+      </div>
+      
+      <div class="row mt-1 mb-3 justify-content-left offset-1">
+        <a href="/contacts">Clear filters</a>
+      </div>  
+
+  <% end %>
+</div>
+
+<div class= "container-flex">
+  <table class ="table">
+    <thead>
+      <tr class="row justify-content-center">
+        <th class="col-md-1">Name</th>
+        <th class="col-md-1">Last name</th>
+        <th class="col-md-2">Phone number</th>
+        <th class="col-md-1">Channel</th>
+        <th class="col-md-2">Email</th>
+        <th class="col-md-1">Organization</th>
+        <th class="col-md-1">Social Networks</th>
+        <th class="col-md-1"></th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @contacts.each do |contact| %>
+        <tr class="row justify-content-center">
```

Comment:
If you *really* want to have `<table>` element, you can move this entire `<tr>` to a view partial, which could make this view template less busy.